hasher-matcher-actioner/hmalib/lambdas/actions/action_evaluator.py

```diff
@@ -52,23 +55,18 @@ def lambda_handler(event, context):
     for sqs_record in event["Records"]:
         # TODO research max # sqs records / lambda_handler invocation
         sqs_record_body = json.loads(sqs_record["body"])
-
```

Comment:
After conversations with Barrett and Dipanjan, it was determined checking for test messages is only applicable to queue reads where the queue message originated as an S3 notification.